zib/StructureDefinitions/zib-Encounter.xml

```diff
@@ -0,0 +1,285 @@
+      <definition value="EncounterSetting describes the manner and setting in which the contact takes place." />
+      <alias value="ContactVorm"/>
+      <binding>
+        <strength value="extensible"/>
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.15.1.2--20240415000000"/>
+      </binding>
+      <mapping>
+        <identity value="zib-encounter-v6.0-2024EN"/>
+        <map value="NL-CM:15.1.19"/>
+        <comment value="EncounterSetting"/>
+      </mapping>
+    </element>
+    <element id="Encounter.type">
+      <path value="Encounter.type"/>
+      <slicing>
+        <discriminator>
+          <type value="value"/>
+          <path value="$this"/>
+        </discriminator>
+        <rules value="open"/>
+      </slicing>
+    </element>
+    <element id="Encounter.type:encounterType">
+      <path value="Encounter.type"/>
+      <sliceName value="encounterType"/>
+      <short value="EncounterType"/>
+      <definition value="The type of encounter is based on the setting of the encounter and the participants present."/>
+      <alias value="ContactType"/>
+      <max value="1"/>
+      <binding>
+        <strength value="required"/>
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.15.1.1--20240415000000"/>
+      </binding>
+      <mapping>
+        <identity value="zib-encounter-v6.0-2024EN"/>
+        <map value="NL-CM:15.1.2"/>
+        <comment value="EncounterType"/>
+      </mapping>
+    </element>
+    <element id="Encounter.type:reason">
+      <path value="Encounter.type"/>
+      <sliceName value="reason"/>
+      <short value="Reason"/>
+      <definition value="Main reason for the encounter within the context of the health problem."/>
+      <alias value="AanleidingContact"/>
+      <max value="1"/>
+      <binding>
+        <strength value="required"/>
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.15.1.3--20240415000000"/>
+      </binding>
+      <mapping>
+        <identity value="zib-encounter-v6.0-2024EN"/>
+        <map value="NL-CM:15.1.20"/>
+        <comment value="Reason"/>
+      </mapping>
+    </element>
+    <element id="Encounter.type:reason.text">
+      <path value="Encounter.type.text"/>
+      <short value="CommentEncounterReason"/>
+      <definition value="Comment on the reason for the encounter, insofar as this cannot be sufficiently expressed in the other elements."/>
+      <alias value="ToelichtingRedenContact"/>
+      <mapping>
+        <identity value="zib-encounter-v6.0-2024EN"/>
+        <map value="NL-CM:15.1.17"/>
+        <comment value="CommentEncounterReason"/>
+      </mapping>
+    </element>
+    <element id="Encounter.participant">
+      <path value="Encounter.participant"/>
+      <slicing>
+        <discriminator>
+          <type value="value"/>
+          <path value="type"/>
+        </discriminator>
+        <discriminator>
```

Comment:
The two slices refer the same profile, so this discriminator doesn't work. The type discriminator should be enough though.